CONTRIBUTING.md

```diff
@@ -16,20 +16,6 @@ If you think we might be out of date with the spec, you can check that by invoki
 
 If you're adding tests to cover something in the spec, use the `@Specification` annotation like you see throughout the test suites.
 
-## End-to-End Tests
```

Comment:
Rather than removing, IMO we should mention that the e2e tests use `InMemoryProvider`. Consider the GO SDK readme for reference - https://github.com/open-feature/go-sdk/blob/main/e2e/README.md